docs/changelog/83665.yaml

```
@@ -0,0 +1,5 @@
+pr: 83665
+summary: "RemoveProcessor updated to support fieldsToKeep"
+area: Infra/Scripting
```

Technically, this falls into the "ingest" category:
```suggestion
area: Ingest
```

---

ok